message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -93,37 +93,37 @@
       <entry value="42004" name="MAV_CMD_FIXED_MAG_CAL">
         <description>Magnetometer calibration based on fixed position
         in earth field given by inclination, declination and intensity.</description>
-        <param index="1">MagDeclinationDegrees.</param>
-        <param index="2">MagInclinationDegrees.</param>
-        <param index="3">MagIntensityMilliGauss.</param>
-        <param index="4">YawDegrees.</param>
+        <param index="1" label="Declination" units="deg">MagDeclinationDegrees.</param>
+        <param index="2" label="Inclination" units="deg">MagInclinationDegrees.</param>
+        <param index="3" label="Intensity" units="mgauss">MagIntensityMilliGauss.</param>
+        <param index="4" label="Yaw" units="deg">YawDegrees.</param>
         <param index="5">Empty.</param>
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
       <entry value="42005" name="MAV_CMD_FIXED_MAG_CAL_FIELD">
-        <description>Magnetometer calibration based on fixed expected field values in milliGauss.</description>
-        <param index="1">FieldX.</param>
-        <param index="2">FieldY.</param>
-        <param index="3">FieldZ.</param>
+        <description>Magnetometer calibration based on fixed expected field values.</description>
+        <param index="1" label="Field X" units="mgauss">FieldX.</param>
+        <param index="2" label="Field Y" units="mgauss">FieldY.</param>
+        <param index="3" label="Field Z" units="mgauss">FieldZ.</param>
         <param index="4">Empty.</param>
         <param index="5">Empty.</param>
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
       <entry value="42424" name="MAV_CMD_DO_START_MAG_CAL">
         <description>Initiate a magnetometer calibration.</description>
-        <param index="1">uint8_t bitmask of magnetometers (0 means all).</param>
-        <param index="2">Automatically retry on failure (0=no retry, 1=retry).</param>
-        <param index="3">Save without user input (0=require input, 1=autosave).</param>
-        <param index="4">Delay (seconds).</param>
-        <param index="5">Autoreboot (0=user reboot, 1=autoreboot).</param>
+        <param index="1" label="Bitmask" minValue="0" maxValue="255" increment="1">Bitmask of magnetometers to calibrate (0 means all).</param>
+        <param index="2" label="Retry on Failure" minValue="0" maxValue="1" increment="1">Automatically retry on failure (0=no retry, 1=retry).</param>
+        <param index="3" label="Autosave" minValue="0" maxValue="1" increment="1">Save without user input (0=require input, 1=autosave).</param>
+        <param index="4" label="Delay" units="s">Delay.</param>
+        <param index="5" label="Autoreboot" minValue="0" maxValue="1" increment="1">Autoreboot (0=user reboot, 1=autoreboot).</param>
```

Comment:
option 0 isn't good but is pre-existing.  It presupposed a reboot is required.  0 should have been "none".

Reply:
are you saying s/user reboot/none/ ?